tutorials/io/background_loading.rst

```diff
@@ -48,22 +48,48 @@ The path will be ``"Enemy.tscn"`` which is located at ``res://Enemy.tscn``.
 
 First, we will start a request to load the resource and connect the button:
 
-::
+.. tabs::
+ .. code-tab:: gdscript
 
     const ENEMY_SCENE_PATH : String = "Enemy.tscn"
 
     func _ready():
         ResourceLoader.load_threaded_request(ENEMY_SCENE_PATH)
         self.pressed.connect(_on_button_pressed)
 
+ .. code-tab:: csharp
+
+    public partial class MyButton : Button
```

Comment:
Since this looks like the complete script file, we should include the `using` statements.

```suggestion
    using Godot;

    public partial class MyButton : Button
```